bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/GmosPrinter.java

```diff
@@ -235,4 +248,23 @@ protected void _printPeakPixelInfo(final scala.Option<ItcCcd> ccd, final GmosSat
                             ccd.get().peakPixelFlux(), ccd.get().adu(), gmosLimit.percentOfLimit(ccd.get().peakPixelFlux()), gmosLimit.limit()));
         }
     }
+
+    public ConfigCreator.ConfigCreatorResult createInstConfig(int numberExposures) {
+        ConfigCreator cc = new ConfigCreator(p);
+        return cc.createGmosConfig(instr, numberExposures);
+    }
+
+    public PlannedTime.ItcOverheadProvider getInst() {
+        if (instr.site().displayName.equals("Gemini North")) {
```

Comment:
you should compare directly with `Site.GN` rather than with the string

Reply:
Fixed.